index.ts

```diff
@@ -1,24 +1,34 @@
+
+console.group("Cargando los resultados...");
 //  1.- Create an array of n unique numbers, from 0 to N - 1, where N is the length of the array.
 //  i: 5
 //  o: [0, 1, 2, 3, 4]
+const crearCajaInicial = (n: number): number[] => {
```

Comment:
return its infered, you don't need to explicit put it there, learn about inference in TS

Reply:
es una forma de representar el array